regparser/grammar/appendix.py

```diff
@@ -30,3 +30,5 @@ def decimalize(characters, name):
 period_upper = decimalize(string.ascii_uppercase, "period_upper")
 period_lower = decimalize(string.ascii_lowercase, "period_lower")
 period_digit = decimalize(string.digits, "period_digit")
+
+roman_upper = decimalize('IVXLCDM', "roman_upper")
```

Comment:
How does this differ from `aI = Word("IVXLCDM").setResultsName("aI")` in [`grammar/appendix.py`](https://github.com/cfpb/regulations-parser/blob/master/regparser/grammar/appendix.py#L17)? 

Apart from being much clearer named, that is.